src/layer/VideoOverlay.js

```diff
@@ -39,8 +39,8 @@ export var VideoOverlay = ImageOverlay.extend({
 		var wasElementSupplied = this._url.tagName === 'VIDEO';
 		var vid = this._image = wasElementSupplied ? this._url : DomUtil.create('video');
 
-		vid.class = vid.class || '';
-		vid.class += 'leaflet-image-layer ' + (this._zoomAnimated ? 'leaflet-zoom-animated' : '');
+		vid.className = vid.className || '';
+		vid.className += 'leaflet-image-layer ' + (this._zoomAnimated ? 'leaflet-zoom-animated' : '');

```

Comment:
seems we would also need to have a space inserted here, changing the line to:

```js
vid.className += ' leaflet-image-layer ' + (this._zoomAnimated ? 'leaflet-zoom-animated' : '');
```

Otherwise, if `className` is specified, it will be concatenated with `leaflet-image-layer`, a class which most certainly will not exist.